### PR TITLE
Send dockable false by iframe

### PR DESCRIPTION
**Pull request by guzzur**
## Context

Explain how we got here and what this PR will contribute once merged

## Documents

List of related documents

## Tests

List of implemented tests

## Screenshots

Add screenshots before and after

### CODE REVIEW

File: src/strigo.sdk.ts [hunk 1]
```diff
@@ -6,9 +6,9 @@ import { Strigo } from './strigo';
 window.Strigo = Strigo;
 
 if (isRecordingMode()) {
-  Logger.info('Strigo recorder mode')
+  Logger.info('Strigo recorder mode');
   const development = urlTools.isDevelopment();
   window.Strigo.assessmentRecorder(development);
-} else {
+} else if (!urlTools.isExtension()) {
```
Comment (guzzur) [inline src/strigo.sdk.ts:12]: @hidday this PR will not be merged, but please take this change to yours.
What is practically does is not initing the sdk when is run by the extension.
If you've done this in a different way, feel free to skip.

@ZobelOmry @DvirMalka FYI

Comment (hidday) [reply src/strigo.sdk.ts:12]: Why we are avoiding the "init" call for non-extension?

Comment (hidday) [reply src/strigo.sdk.ts:12]: I think our last changes (the Felix-time-wasting session we had), should address it.

Comment (hidday) [reply src/strigo.sdk.ts:12]: So its like -> always inject -> always init -> get config from strigo-app -> setup and open only when its required (the correct domain)

Comment (hidday) [reply src/strigo.sdk.ts:12]: For non-extension flow I would start the same: always inject -> always init 
then you can access your server to generate a token -> then you can trigger the setup/open flow

Comment (guzzur) [reply src/strigo.sdk.ts:12]: > I think our last changes (the Felix-time-wasting session we had), should address it.

Zdayen and do what you believe

> For non-extension flow I would start the same: always inject -> always init
then you can access your server to generate a token -> then you can trigger the setup/open flow

Sounds good

**Pull request closed without merging**